### DISCUSSION TITLE
Penna is interested in joining

### DISCUSSION THREAD

**Discussion by hkupty** [commonhaus/foundation#139] [Joining Commonhaus]
### Project information

- **Project name**: Penna
- **Project website**: https://hkupty.github.io/penna/
- **Code repository**: https://github.com/hkupty/penna
- **License**: [MIT License](https://github.com/hkupty/penna/blob/dev/0.8/LICENSE)

### Describe your project

I believe Penna is a good candidate because it is a small, focused project.

It is a JDK21, vanilla java SLF4J implementation that sets to solve a common use case: writing json logs to stdout. It is aimed at helping modern cloud usage in kubernetes and containerized environments.

It's benefits over other established alternatives are simpler setup, better performance and k8s friendly config (live-updatable yaml file). Given its relatively small surface area, it might as well be safer.

The project needs support and visibility to break through the status quo and reach bigger audiences, as a single individual can't make enough noise alone. It is completely open source and not sponsored by any company.

**Comment by ebullient** [comment]
Please use this discussion (not necessarily this sub-thread) to ask any other questions you may have.

- [ ] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting). You may have seen some PRs go through as I work with the current EGC to create a record of approval with the newly established EGC) to ratify them and you're welcome to weigh in on these.
- [ ] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [ ] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)
